python/llm/src/ipex_llm/utils/ipex_importer.py

```diff
@@ -56,12 +56,22 @@ def import_ipex(self):
         Raises ImportError if failed
         """
         if self.is_xpu_version_installed():
+            ipex_imported = False
+            try:
+                distribution('intel_extension_for_pytorch')
+                ipex_importer = True
+            except PackageNotFoundError:
+                pass
+            if ipex_imported:
+                logging.warning("IPEX-LLM will automatically import Intel Extension for PyTorch.",
+                                "Please avoid import Intel Extension for PyTorch manually!")
```

Comment:
Report error "Please import ipex-llm before import Intel Extension for PyTorch"

Reply:
Done